src/main/java/seedu/address/model/tag/Tag.java

```diff
@@ -10,8 +10,9 @@
 public class Tag {
 
     public static final String MESSAGE_CONSTRAINTS = "Tags names should be alphanumeric and can have spaces, but "
-            + "cannot be empty";
-    public static final String VALIDATION_REGEX = "\\p{Alnum}[\\p{Alnum}\\p{Zs}]*";
+            + "cannot be empty or exceed 100 characters";
+    public static final String MAX_TAGS_CONSTRAINTS = "A person should only have up to 10 tags.";
+    public static final String VALIDATION_REGEX = "\\p{Alnum}[\\p{Alnum}\\p{Zs}]{0,99}";
```

Comment:
Would {0,99} regex mean allowing 0-99 characters instead of 1-100 characters? Should it be {1, 100} instead?

Reply:
The Alnum in front is already 1 character, so on top of that it should be 0 to 99 characters. This should be correct.